dev-tools/packaging/packages.yml

```diff
@@ -157,6 +157,19 @@ shared:
         template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/windows/uninstall-service.ps1.tmpl'
         mode: 0755
 
+  - &docker_spec
+    <<: *binary_spec
+    extra_vars:
+      from: 'centos:7'
+      user: '{{ .BeatName }}'
+      repository: 'docker.elastic.co/beats'
+      linux_capabilities: ''
```

Comment:
feature request: something like `expose_port` - and then use like [this](https://github.com/elastic/apm-server-docker/blob/b1b58e119edd106eb2882fb5e3db6e2b9d9ef734/templates/Dockerfile.j2#L37)

Reply:
Done, `expose_ports` can be a comma separated list of ports.